templates/base.html

```diff
@@ -16,47 +16,61 @@
     <link rel="stylesheet" href="static/style.css" />
```

Comment:
This should be `href="/static/style.css"` - with a leading `/`, because on pages like `/station/results` it will take everything from the root and append the `static/style.css`, which produces a 404.